mythril/laser/ethereum/svm.py

```diff
@@ -527,6 +490,10 @@ def register_laser_hooks(self, hook_type: str, hook: Callable):
             self._start_sym_exec_hooks.append(hook)
         elif hook_type == "stop_sym_exec":
             self._stop_sym_exec_hooks.append(hook)
+        elif hook_type == "start_sym_trans":
+            self._start_sym_trans_hooks.append(hook)
+        elif hook_type == "stop_sym_trans":
```

Comment:
As this part might  increase in the future, we can make a static dictionary of strings and functions and directly use it, so each hook access would be in O(1) if we move the static dictionary into the svm initialization. but it is a negligible optimization, anyway using a dictionary in these cases is a bit more clean, WDYT?

Reply:
I agree, was planning on doing something similar. But not in this pr 😅